festim/problem.py

```diff
@@ -121,6 +121,7 @@ def run(self):
             # Solve steady-state
             self.solver.solve(self.u)
             self.post_processing()
+        self.progress_bar.close()
```

Comment:
@kaelyndunnell this has to move to the `if self.settings.transient` part since progress bar is only created for transient cases. This is why the CI is failing on this PR now